.github/workflows/gh-pages.yml

```
@@ -33,10 +33,16 @@ jobs:
 
       - name: Deploy
         uses: peaceiris/actions-gh-pages@v3
```

Would you please use sha so the action does not accidentally hijacked? We pass deploy key to the action, so it is better play safe here 

```suggestion
        uses: peaceiris/actions-gh-pages@068dc23d9710f1ba62e86896f84735d869951305 # v3.8.0
```